packages/node/__tests__/integration.spec.ts

```diff
@@ -696,6 +700,109 @@ describe('Incremental Delivery', () => {
   })
 })
 
+function md5File(path: string) {
+  return new Promise((resolve, reject) => {
+    const output = crypto.createHash('md5')
+    const input = fs.createReadStream(path)
+
+    input.on('error', (err) => {
+      reject(err)
+    })
+
+    output.once('readable', () => {
+      resolve(output.read().toString('hex'))
+    })
+
+    input.pipe(output)
+  })
+}
+
+describe.only('file uploads', () => {
```

Comment:
We already have some upload tests here. Maybe we can combine these.
https://github.com/dotansimha/graphql-yoga/blob/a771f1359dd45c6a83e49b7285d64dd8ab781509/packages/node/__tests__/integration.spec.ts#L559